google/cloud/storage/grpc_plugin_test.cc

```diff
@@ -91,18 +91,19 @@ TEST(GrpcPluginTest, NoneConfigCreatesCurl) {
               ElementsAre("RestStub", "StorageConnectionImpl"));
 }
 
-TEST(GrpcPluginTest, MediaConfigCreatesHybrid) {
+TEST(GrpcPluginTest, MediaConfigCreatesMetadata) {
   // Explicitly disable logging, which may be enabled by our CI builds.
   auto logging =
       ScopedEnvironment("CLOUD_STORAGE_ENABLE_TRACING", absl::nullopt);
+  // We used to support a "media" configuration where only media operations
```

Comment:
Nit: Do we have to say what we used to do?

Reply:
Removed, and refactored the test some more.